src/routes.js

```diff
@@ -12,11 +13,11 @@ export const ROUTES_PATHS = {
     LOGOUT: '/',
     USER_PROFILE: '/user-profile',
     USER_RECIPES: '/user-profile/recipes',
-    USER_RECIPES_ADD: '/user-profile/recipes/new',
+    USER_RECIPES_EDIT: '/user-profile/recipes/edit',
```

Comment:
we should keep two urls,
you can just add another route with the same component but changed url. It will prevent us from doing additional work if we separate upsertComponent in the future